sources/packages/web/src/services/EducationProgramService.ts

```diff
@@ -9,10 +9,21 @@ export class EducationProgramService {
     return this.instance || (this.instance = new this());
   }
 
-  public async createProgram(data: any): Promise<void> {
+  public async getProgram(programId: number): Promise<any> {
```

Comment:
instead of promise `any` can we create a interface

Reply:
Yes, I added, but right now it is not gonna pass in Sonar Cloud due to a duplication constraint.
Right now, we are trying to figure out how to configure the Sonal Cloud to ignore it.
The other way would be to share the DTOs between API and web, which we are not doing right now.
I don't like any, but it is the only way to have the PR passing the gates right now.